psi4/src/psi4/fisapt/fisapt.cc

```diff
@@ -1441,11 +2461,11 @@ void FISAPT::exch() {
     for (int k = 0; k < Exch10_2M_terms.size(); k++) {
         Exch10_2M += Exch10_2M_terms[k];
     }
-    // for (int k = 0; k < Exch10_2M_terms.size(); k++) {
-    //    outfile->Printf("    Exch10(S^2) (%1d)     = %18.12lf [Eh]\n",k+1,Exch10_2M_terms[k]);
-    //}
+    for (int k = 0; k < Exch10_2M_terms.size(); k++) {
```

Comment:
Just to be clear, the things here were uncommented and left uncommented intentionally?

Reply:
Yup, that was debug printing that is not needed anymore. Good catch!